src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -532,7 +579,9 @@ static bool checkHelpLink(string helpLink)
             string? category,
             string? customTag,
             SortedList<string, DiagnosticDescriptor> sortedRulesById,
-            string analyzerPackageName)
+            string analyzerPackageName,
+            bool validateOnly,
+            ref bool validationResult)
```

Comment:
Also it looks like this parameter is never used.

Reply:
It's used in line 604